DirectProgramming/DPC++/SpectralMethods/DiscreteCosineTransform/DCT.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30011.22
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "DCT", "msvs\DCT.vcxproj", "{B2789FC1-D75F-4C6E-BF93-05BE084FBC79}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Intel Performance Test|x64 = Intel Performance Test|x64
+		Intel Release|x64 = Intel Release|x64
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{B2789FC1-D75F-4C6E-BF93-05BE084FBC79}.Intel Performance Test|x64.ActiveCfg = Intel Performance Test|x64
+		{B2789FC1-D75F-4C6E-BF93-05BE084FBC79}.Intel Performance Test|x64.Build.0 = Intel Performance Test|x64
+		{B2789FC1-D75F-4C6E-BF93-05BE084FBC79}.Intel Release|x64.ActiveCfg = Intel Release|x64
+		{B2789FC1-D75F-4C6E-BF93-05BE084FBC79}.Intel Release|x64.Build.0 = Intel Release|x64
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {55E32F12-32F5-4567-B736-8B38914AAB2E}
+	EndGlobalSection
+EndGlobal
```

Comment:
Should this file be merged? This seems like your personal workspace information.

Reply:
Hey Anjali,

The .sln file is required for my command line build instructions documented in the README.

They require the user to call MSBuild and pass the .sln file as an argument, which allows the program to be build per the configurations set in visual studio.

As far as I know, this file shouldn't pose any issues, unless there's something I'm not aware of?


Side note: My latest commit should have resolved the dpc_common discrepancy. Thank you

Reply:
Okay, thanks for the explanation.